src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -319,7 +353,7 @@ class ContentExplorer extends Component {
 
         if (item.type === TYPE_FOLDER) {
             this.enterFolder(item);
-        } else {
+        } else if (!item.isActionDisabled) {
```

Comment:
what's item.isActionDisabled vs item.isDisabled?

Reply:
actionDisabled means that the item's name can still be clicked on to see what is inside the folder. But it's checkbox cannot be interacted with and it cannot be selected.